samples/aspnetcore/Abstractions/Abstractions.csproj

```diff
@@ -1,11 +1,11 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.Hosting" Version="2.0.0" />
+    <PackageReference Include="Microsoft.AspNetCore.Hosting" Version="2.1.1" />
```

Comment:
I made this sample to target 2.0.0 intentionally because I wanted to show how abstractions can target a lower version of assemblies and let the app lift to a higher version using the `sharedTypes` parameter on PluginLoader. Can you keep this at 2.0.0?